### PR TITLE
Set correct focus for login button

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

File: frontend/src/layouts/GLogin.vue [hunk 1]
```diff
@@ -267,10 +266,18 @@ export default {
...
+      async handler (value) {
+        if (value === 'token') {
+          setDelayedInputFocus(this, 'tokenField')
+        }
+        if (value === 'oidc') {
+          setTimeout(() => {
+            this.$refs.loginButton.$el.focus()
+          }, 0)
+        }
+      },
```
Comment (holgerkoser) [inline frontend/src/layouts/GLogin.vue:279]:
```suggestion
      handler (value) {
        if (value === 'token') {
          setDelayedInputFocus(this, 'tokenField')
        }
        if (value === 'oidc') {
          setDelayedInputFocus(this, 'loginButton.$el', {
            delay: 0,
            noSelect: true,
          })
        }
      },
```

Comment (holgerkoser) [reply frontend/src/layouts/GLogin.vue:279]: And adjust setInputFocus:
```js
export function setInputFocus (vm, fieldName, options) {
  if (typeof fieldName === 'string') {
    vm = get(vm.$refs, fieldName) // ------ this line
  } else {
    vm = unref(vm)
    options = fieldName
  }
```

Comment (grolu) [reply frontend/src/layouts/GLogin.vue]: ok, done

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**